woocommerce/google-listings-and-ads
# Use @wordpress/prettier-config package

### Changes proposed in this Pull Request:

Change the Prettier config setup, by using https://www.npmjs.com/package/@wordpress/prettier-config package and  following the documentation in https://www.npmjs.com/package/@woocommerce/eslint-plugin:

> If you want to use prettier in your code editor, you'll need to create a `.prettierrc.js` file at the root of your project with the following:
>
> ```
> module.exports = require("@wordpress/prettier-config");
> ```

https://www.npmjs.com/package/@wordpress/prettier-config is essentially the same as our current `.prettierrc` file, so there should be no change to our code editing and formatting. See the config below:

https://github.com/WordPress/gutenberg/blob/0d34c1c46a49e585e2b7bea1df9821802c4708a8/packages/prettier-config/lib/index.js#L13-L24

### Detailed test instructions:

1. Checkout the branch.
2. Run `npm ci`.
3. Make sure linting and formatting still work the same in your editor (e.g. "auto format on save" feature in VS Code). You might have to restart your editor.

### Changelog entry

> Tweak - Use @wordpress/prettier-config package.
